docsrc/conf.py

```diff
@@ -160,7 +161,7 @@
 # -- Options for HTMLHelp output ------------------------------------------
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = project + "doc"
```

Comment:
Is there a reason for not using the `project` variable here and elsewhere in `conf.py` (line 187, 195, etc...)? `project` is defined above in line 51 and can be used throughout `conf.py`. This allows for robust programming.